src/giskardpy/utils.py

```diff
@@ -165,6 +166,36 @@ def cylinder_surface(r, h):
 #             return True
 #     return False
 
+def calculate_waypoint2D(target, origin, distance):
```

Comment:
If you remove the set method from the interface, this can be removed too, right?

Reply:
This method is used, but it makes more sense to move that to suturo_manipulation, where it is used

Reply:
Is moved